### PR TITLE
Showcase Redesing 

### PR DESCRIPTION
**Pull request by GomezMimo**
Staging link: https://huge-flutter-dev-staging.firebaseapp.com/showcase

### CODE REVIEW

File: src/_assets/css/_variables.scss [hunk 1]
```diff
@@ -12,7 +12,9 @@ $site-color-nav-links: #6E7274;
 $site-color-body: #4A4A4A;
 $site-color-body-light: rgba(74/255, 74/255, 74/255, .8);
 $site-color-footer: #303c42;
+$site-color-paragraph: #555;
 $site-color-primary: $flutter-color-blue-500;
+$site-color-title: #212C34;
```
Comment (1Iv4N) [inline src/_assets/css/_variables.scss]: It looks like we're using this color only in the showcase page. Let's move it to the _showcase.scss file.

Comment (1Iv4N) [inline src/_assets/css/_variables.scss]: It looks like we're using this color only in the showcase page. Let's move it to the _showcase.scss file.

**Review by 1Iv4N — CHANGES_REQUESTED** [review]
Hey Juan, let's do the following changes:
- Please optimize all SVG files. It will help improve performance.
- Revert the last 2 commits (5e26d354, 30185210) it looks like you're reverting some updates made on the upstream repository.

**Comment by GomezMimo** [comment]
Comments done. @1Iv4N

File: src/_assets/css/_variables.scss (hunk 1 above)
Comment (GomezMimo) [reply src/_assets/css/_variables.scss]: Done

**Review by 1Iv4N — APPROVED** [review]

**Pull request merged**